### DISCUSSION TITLE
Unable to make a test request from UI when using vue component library from npm in local project

### DISCUSSION THREAD

**Discussion by FotieMConstant** [scalar/scalar#351] [Q&A] [answered]
Hi, i am having this issue where i can't seem to send a request from the ui. each time i do, it gives me an error as seen below:

<img width="1280" alt="image" src="https://github.com/scalar/scalar/assets/42372656/ae63be4f-032f-43e7-8eba-b29549e0d507">

Now, i am using the vue component library from npm.

However, when i check the network tab and then, response. i see that the response came in but it doesn't get displayed. not sure why though.

<img width="1271" alt="image" src="https://github.com/scalar/scalar/assets/42372656/480eb88d-1653-4fed-919f-a55ff0eb6e69">

**Comment by marclave** [comment]
@FotieMConstant thanks for posting this!

Sorry I missed this today, will investigate and get back to you :)

**Comment by marclave** [comment]
@FotieMConstant I can't seem to reproduce this, are you able to share your swagger file?

I tested out the endpoint with our client/proxy and it seemed to be working well

<img width="1496" alt="image" src="https://github.com/scalar/scalar/assets/6176314/7e406da1-f2dd-4445-9895-b54d903deeeb">

**Reply by FotieMConstant** [reply]
Hey @marclave, thanks for the reply. I’m using the vuejs component library. Could you try it with the component library, will share my swagger file shortly

**Reply by FotieMConstant** [reply]
Here is my swagger file: https://github.com/FotieMConstant/lanadoc/blob/main/public/resource/openapi-spec-sample.yaml

**Reply by marclave** [reply]
ill do a fresh vue app and try it out :)

**Reply by FotieMConstant** [reply]
Thanks @marclave 

Also, is there a sort of project timeline, to-do or something for scalar? How are they assigned to contributors? It's an amazing tool and i'd love to contribute and help make it even better.

**Reply by marclave** [reply]
@FotieMConstant I did a fresh repo from latest vue and it worked fine for me, could you share with me the repo if it's public and I can do a clone and try to fix it.

Also what browser are you using?

<img width="1080" alt="image" src="https://github.com/scalar/scalar/assets/6176314/71e84f4f-61d8-420f-9434-1a56f332795b">

<img width="1511" alt="image" src="https://github.com/scalar/scalar/assets/6176314/0c33cf74-ece9-408e-8a5c-22af798aaed8">

**Reply by marclave** [reply]
> Also, is there a sort of project timeline, to-do or something for scalar? How are they assigned to contributors? It's an amazing tool and i'd love to contribute and help make it even better.

@hanspagel and I need to sit down, make an official roadmap and plan this out. Your kind words are incredibly appreciated and your contributions would be everything! ✨

**Reply by FotieMConstant** [reply]
Yeah, actually i am not passing the swagger file url from github or an external url, i am passing it from the local server in the public folder. the swagger file is in the local host public folder. that is where i am getting it from. Thing is, the documentation generates pretty well but i am unable to make calls from within the ui.

here is the link to the project with the vue component library installed

link: https://github.com/FotieMConstant/lanadoc

PS: Maybe there is something i am doing wrong?

**Reply by FotieMConstant** [reply]
Hey @marclave any update on this? i can't seem to get it working either. it doesn't equally work when i pass in an actual url as a prop to the specUrl as seen on the screenshoot you added above. Not sure why though.

PS: I am using chrome

**Reply by hanspagel** [reply, marked as answer]
Thanks for providing the repo! I can reproduce it there. I can make it work with a request proxy:

```vue
<ApiReference
  :specUrl="apiSpecUrl"
  proxyUrl="https://api.scalar.com/request-proxy"
/>
```

Looks like I’ve done a bad job testing it without a proxy, I’ve created an issue: https://github.com/scalar/scalar/issues/361

**Reply by FotieMConstant** [reply]
Hey @hanspagel thanks for this, I’ll try it out. Also, i can take on the issue you opened if no one is currently on it?

**Reply by marclave** [reply]
@FotieMConstant you can absolutely take on that issue :)
